### DISCUSSION TITLE
ingress controller

### DISCUSSION THREAD

**Discussion by kzzz1** [DefectDojo/django-DefectDojo#10767] [Q&A]
Hi,
we have setup an ingress controller in kubernetes and are using helm charts for the install.
Is there a place where I can find the minimum version compatibility of the ingress controller (ingress-nginx) with defectdojo? 

Thank you

**Comment by mtesauro** [comment]
FYI: There's generally more response plus back and forth conversations on #defectdojo on OWASP's Slack. You can find out how to join at https://github.com/DefectDojo/django-DefectDojo?tab=readme-ov-file#community-getting-involved-and-updates

HTH
